backend/lcfs/db/models/NotificationChannelSubscription.py

```diff
@@ -0,0 +1,19 @@
+from lcfs.db.base import BaseModel, Auditable
+from sqlalchemy import Column, Integer, ForeignKey, Boolean
+from sqlalchemy.orm import relationship
+
+class NotificationChannelSubscription(BaseModel, Auditable):
+     __tablename__ = 'notification_channel_subscription'
+     __table_args__ = {'comment': "Represents a user's subscription to notification events"}
+
+     id = Column(Integer, primary_key=True, autoincrement=True)
+
+     is_enabled = Column(Boolean, default=False)
+
+     user_id = Column(Integer, ForeignKey('user.id'))
+     channel_id = Column(Integer, ForeignKey('notification_channel.id'))
```

Comment:
these foreign keys will have to be updated, notification_channel.notification_channel_id
a bit redundant but for database standards we'll have to bear it.